miden-tx/src/testing/utils.rs

```diff
@@ -1,93 +1,8 @@
-use alloc::vec::Vec;
-
 use miden_lib::transaction::memory;
-#[cfg(not(target_family = "wasm"))]
-use miden_lib::transaction::TransactionKernel;
-#[cfg(feature = "std")]
-use miden_objects::Felt;
-use miden_objects::{
-    accounts::{
-        account_id::testing::ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_OFF_CHAIN, Account,
-        AccountCode, AccountDelta,
-    },
-    notes::Note,
-    testing::{
-        block::{MockChain, MockChainBuilder},
-        notes::AssetPreservationStatus,
-    },
-    transaction::{ExecutedTransaction, OutputNote, OutputNotes, TransactionOutputs},
-    vm::CodeBlock,
-    FieldElement,
-};
-use vm_processor::{AdviceInputs, Operation, Program, ZERO};
-
-use super::TransactionContextBuilder;
 
 // TEST HELPERS
 // ================================================================================================
 
 pub fn consumed_note_data_ptr(note_idx: u32) -> memory::MemoryAddress {
     memory::CONSUMED_NOTE_DATA_SECTION_OFFSET + note_idx * memory::NOTE_MEM_SIZE
 }
-
-pub fn mock_executed_tx(asset_preservation: AssetPreservationStatus) -> ExecutedTransaction {
```

Comment:
This was a very weird implementation, the dummy program `push.0 drop` had nothing to do with the actual tests, which always had custom code. also most tests didn't need the final account state (only one used it).

so in order to make the test setup and the actual test consistent with one another, I removed this in favor of just using the context. It is less code, less indirection, less inconsistencies, and the same results.